client/signup/config/flows-pure.js

```diff
@@ -438,6 +438,16 @@ export function generateFlows( {
 			description: 'A flow for DIFM Lite leads',
 			lastModified: '2021-09-30',
 		},
+		{
+			name: 'woocommerce-install',
+			pageTitle: translate( 'Add WooCommerce to your site' ),
+			steps: [ 'confirm', 'transfer', 'install', 'complete' ],
```

Comment:
You probably already know this, but step names share a global namespace. `complete` is a fairly generic name so if you take it, it means other flows can't use it. This is more just an FYI. I think it's first-come-first-serve, otherwise we'll end up reserving all of the good step names and never actually using them.

Reply:
> You probably already know this
Hah yea, figured, you snooze you lose. The url's look a lot better without namespacing at each path level.

Reply:
The framework does have a sub-step system (disclaimer: I've never used it) that could be used instead. Long urls though: `/start/wocommerce-install/some-step-name/confirm`

The domain step uses it for sub-steps to do with using an existing domain if you're interested.